spinnman/transceiver.py

```diff
@@ -1253,6 +1253,68 @@ def get_iobuf(self, core_subsets=None):
         process = ReadIOBufProcess(self._scamp_connection_selector)
         return process.read_iobuf(self._iobuf_size, core_subsets)
 
+    def set_watch_dog_mode_at_chip(self, watch_dog_mode, chip_x, chip_y):
+        """ changes the chips watchdog parameter to watch_dog_mode
+
+        :param watch_dog_mode: the mode / value to change the watchdog value\
```

Comment:
Probably needs a better explanation i.e. when bool and True, sets to default value, when bool and False, disables watchdog, when int, sets to the given value